### DISCUSSION TITLE
ECIS Series Example

### DISCUSSION THREAD

**Discussion by MusaabKh** [SmartDataAnalytics/OpenResearch#194] [General]
ECIS is a series which all events correspond to WikiCFP.
- https://confident.dbis.rwth-aachen.de/or/index.php/ECIS
- [PTP details of ECIS example](http://ptp.bitplan.com/parse?titles=ECIS+2020%0D%0AECIS+2019%0D%0AECIS+2018%0D%0AECIS+2017%0D%0AECIS+2016%0D%0AECIS+2015%0D%0AECIS+2014%0D%0AECIS+2013%0D%0AECIS+2012%0D%0AECIS+2011%0D%0AECIS+2010%0D%0AECIS+2009%0D%0AECIS+2008%0D%0AECIS+2007%0D%0AECIS+2006%0D%0AECIS+2005%0D%0AECIS+2004%0D%0AECIS+2003%0D%0AECIS+2002%0D%0AECIS+2001%0D%0AECIS+2000%0D%0AECIS+1999%0D%0AECIS+1998%0D%0AECIS+1997%0D%0AECIS+1996%0D%0AECIS+1995%0D%0AECIS+1994%0D%0AECIS+1993&examples=example1&format=html&metadebug=on)

**Comment by WolfgangFahl** [comment]
http://www.wikicfp.com/cfp/program?id=808

**Comment by WolfgangFahl** [comment]
Location, homepage and date of conference may be looked up via GND ID we need to find a way to apply #183 
e.g. 
[ECIS 2019](https://confident.dbis.rwth-aachen.de/or/index.php?title=ECIS_2019)

https://lobid.org/gnd/1198503130.json
```json
{
  "geographicAreaCode": [{
    "id": "https://d-nb.info/standards/vocab/gnd/geographic-area-code#XA-SE",
    "label": "Schweden"
  }],
  "dateOfConferenceOrEvent": ["08.06.2019-14.06.2019"],
  "describedBy": {
    "id": "https://d-nb.info/gnd/1198503130/about",
    "license": {
      "id": "http://creativecommons.org/publicdomain/zero/1.0/",
      "label": "http://creativecommons.org/publicdomain/zero/1.0/"
    },
    "dateModified": "2019-11-01T09:44:11.000"
  },
  "gndIdentifier": "1198503130",
  "id": "https://d-nb.info/gnd/1198503130",
  "preferredName": "European Conference on Information Systems (27. : 2019 : Stockholm; Uppsala)",
  "type": ["AuthorityResource", "ConferenceOrEvent"],
  "variantName": ["ECIS (27. : 2019 : Stockholm; Uppsala)"],
  "@context": "http://lobid.org/gnd/context.jsonld",
  "homepage": [{
    "id": "http://ecis2019.eu/",
    "label": "http://ecis2019.eu/"
  }],
  "placeOfConferenceOrEvent": [{
    "id": "https://d-nb.info/gnd/4078659-6",
    "label": "Uppsala"
  }, {
    "id": "https://d-nb.info/gnd/4057648-6",
    "label": "Stockholm"
  }],
  "sameAs": [{
    "id": "http://viaf.org/viaf/4696157282930403640000",
    "collection": {
      "id": "http://www.wikidata.org/entity/Q54919",
      "abbr": "VIAF",
      "publisher": "OCLC",
      "icon": "http://viaf.org/viaf/images/viaf.ico",
      "name": "Virtual International Authority File (VIAF)"
    }
  }, {
    "collection": {
      "abbr": "DNB",
      "name": "Gemeinsame Normdatei (GND) im Katalog der Deutschen Nationalbibliothek",
      "publisher": "Deutsche Nationalbibliothek",
      "icon": "https://www.dnb.de/SiteGlobals/Frontend/DNBWeb/Images/favicon.png?__blob=normal&v=4",
      "id": "http://www.wikidata.org/entity/Q36578"
    },
    "id": "https://d-nb.info/gnd/1198503130/about"
  }]
}
```

**Reply by WolfgangFahl** [reply]
```sql
select * from event_gnd 
where eventId="1198503130"

homepage	eventId	variant	title	date	areaCode	place	url	source	topic	acronym	lookupAcronym
http://ecis2019.eu/	1198503130	ECIS (27. : 2019 : Stockholm; Uppsala)	European Conference on Information Systems (27. : 2019 : Stockholm; Uppsala)	08.06.2019-14.06.2019	https://d-nb.info/standards/vocab/gnd/geographic-area-code#XA-SE	https://d-nb.info/gnd/4078659-6	https://d-nb.info/gnd/1198503130	gnd	(null)	(null)	(null)
```

**Comment by WolfgangFahl** [comment]
```
curl "https://lobid.org/gnd/search?q=ECIS"
```

**Comment by WolfgangFahl** [comment]
https://dblp.org/db/conf/ecis/index.html

**Comment by WolfgangFahl** [comment]
```sql
select * from event_gnd 
where variant like "ECIS%"
```
|homepage|eventId|variant|title|date|areaCode|place|url|source|year|topic|startDate|endDate|acronym|lookupAcronym|
|-----------|-------|-------|----|----|----------|-----|---|-------|----|-----|---------|---------|--------|----------------|
|http://ecis2018.eu/|1198501588|ECIS (26. : 2018 : Portsmouth)|European Conference on Information Systems (26. : 2018 : Portsmouth)|23.06.2018-28.06.2018|https://d-nb.info/standards/vocab/gnd/geographic-area-code#XA-GB|https://d-nb.info/gnd/4046835-5|https://d-nb.info/gnd/1198501588|gnd|2018|(null)|2018-06-23|2018-06-28|(null)|(null)|
|http://ecis2019.eu/|1198503130|ECIS (27. : 2019 : Stockholm; Uppsala)|European Conference on Information Systems (27. : 2019 : Stockholm; Uppsala)|08.06.2019-14.06.2019|https://d-nb.info/standards/vocab/gnd/geographic-area-code#XA-SE|https://d-nb.info/gnd/4078659-6|https://d-nb.info/gnd/1198503130|gnd|2019|(null)|2019-06-08|2019-06-14|(null)|(null)|
